docs/pages/application-access/guides/connecting-apps.mdx

```diff
@@ -3,7 +3,7 @@ title: Use Teleport for Application Access
 description: How to configure Teleport for Application Access.
 ---
 
-# Connecting Web Applications
+# Web Applications Access
```

Comment:
I think we can remove this H1 and change the title to "Web Application Access". The purple header at the top of the page is already an H1.